certsubmitter/certsubmitter.go

```diff
@@ -124,6 +124,12 @@ func checkSCT(sct *ct.SignedCertificateTimestamp, chain []*x509.Certificate, sv
 		errs = append(errs, &errors.SignatureVerificationError{Err: err})
 	}
 
+	// Verify SCT timestamp.
```

Comment:
nit, stuttering: "Signed Certificate Timestamp" timestamp. I don't see a way around this, and I also understand that's how the struct is built and named.

Reply:
oh well. Naming is not our strongest skill.